[IMP] CRM: Partner autocomplete update (#17126)
odoo/documentation: content/applications/sales/crm/optimize/partner_autocomplete.rst

====================
Partner autocomplete
====================

.. |IAP| replace:: :abbr:`IAP (In-App Purchase)`

*Partner autocomplete* is an In-App Purchase (|IAP|) service that enriches business contacts with

> information and data about that business. In any app or module where a Contacts form is encountered,

Reviewer: ```suggestion
information and data about that business. In any app or module where a *Contacts* form is encountered,
```

a business's name can be entered into the :guilabel:`Customer` field (`partner_id` technical field)
and a suggested company can be chosen from the drop-down menu. With *partner autocomplete*, valuable
information and hard-to-find data about companies of all sizes is just a click away.

The information provided by partner autocomplete can include general information about the business
(including full business name and logo), :guilabel:`Phone` number, :guilabel:`Email`, :guilabel:`Tax

> ID`, address and UNSPSC activities as :guilabel:`Tags`.

Reviewer: ```suggestion
ID`, address, and UNSPSC activities as :guilabel:`Tags`.
```

When getting a company's contact information make sure to be aware of the latest EU regulations. For
more information about General Data Protection Regulation refer to: `Odoo GDPR
<http://odoo.com/gdpr>`_. In Odoo, individual contact information cannot be searched for with the
partner autocomplete feature.

.. important::
   Partner autocomplete only works for newly created Contacts. Businesses that already exist in the
   **Contacts** app cannot be enriched.

Configuration
=============

Go to :menuselection:`Settings app --> Contacts section`. If the :guilabel:`Partner Autocomplete`
feature isn't active, activate it by ticking the checkbox beside it and clicking :guilabel:`Save`.

.. image:: partner_autocomplete/settings-partner-autocomplete.png
   :alt: View of settings page and the activations of the feature in Odoo.

Enrich contacts with corporate data
===================================

With *partner autocomplete* enabled, Odoo reveals a large drop-down menu of potential match

> suggestions whenever a user is entering the name of a new contact. If one of the suggestions is

Reviewer: ```suggestion
suggestions based on the name entered in the new contact form. If one of the suggestions is
```

selected, the contact is then populated with relevant data.

> For example, after typing `Odoo`, the following options pop up:

Reviewer: ```suggestion
For example, after typing `Odoo`, the following options appear in the drop-down menu:
```

Reviewer: imo it doesn't really feel like a pop-up

.. image:: partner_autocomplete/odoo-autocomplete.png

>    :alt: Creating a new contact in Odoo

Reviewer 2: For accessibility, it's not part of our style to refer to content within images in our text. My general approach is  This was a mistake in the original doc that didn't get addressed. What I recommend is summarizing what's in the image in line 42, and writing a more descriptive alt text (:

Pricing
=======

As an :abbr:`IAP (In-App Purchase)` service, *Partner Autocomplete* using requires prepaid credits.
Each request consumes one credit. Enterprise Odoo users with a valid subscription get free credits

> to test |IAP| features before deciding to purchase more credits for the database. This includes demo

Reviewer: optional suggestion, the original language feels unwieldy

and training databases, educational databases, and one-app-free databases.

To buy credits, confirm that :guilabel:`Partner Autocomplete` is enabled as above, then navigate to
:menuselection:`Settings app --> Contacts section`. Then, click :guilabel:`Manage Service & Buy
Credits` under :guilabel:`Partner Autocomplete`. On the :guilabel:`Partner Autocomplete` page, click

> :guilabel:`Buy Credit`. On the |IAP| page that loads, choose a desired package and click

Reviewer: ```suggestion
Credits` under :guilabel:`Partner Autocomplete`. On the :guilabel:`Partner Autocomplete` page, click
:guilabel:`Buy Credit` and the |IAP| page loads. Choose a package and click
```
